apps/relayer/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Copy source code and build the project
+FROM node:20-alpine AS builder
+
+WORKDIR /builder
+
+COPY . .
+
+RUN npm i -g pnpm@9
+RUN pnpm install
+RUN pnpm run build
+
+# Create image by copying build artifacts
+FROM node:20-alpine AS runner
+RUN npm i -g pnpm@9
+
+RUN mkdir -p ~/rapidsnark/build; \
```

Comment:
what is rapidnsnark used for in the relayer?

Reply:
Yes, it's only for tests and I can remove it.